test/functional/mempool_updatefromblock.py

```diff
@@ -17,7 +17,7 @@
 class MempoolUpdateFromBlockTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
-        self.extra_args = [['-limitdescendantsize=1000', '-limitancestorsize=1000']]
+        self.extra_args = [['-limitdescendantsize=1000', '-limitancestorsize=1000', '-limitancestorcount=100']]
```

Comment:
Is this change necessary? I'm currently updating the functional test for MiniWallet compatiability

Reply:
yes

Reply:
I see that you are increasing the node's allowed `limitancestorcount` here, but I don't think it's being utilized? Bc the test params are hard-coded at L119 `self.transaction_graph_test(size=100, n_tx_to_mine=[25, 50, 75])`, and don't make use of the increased limit here. So you'd need to update that as well, right?

Reply:
Did this get addressed?

Reply:
Yes, without this param the old limit wasn't being fully tested since it defaults to 25, with the new code we test the limit up until 75 -- the 100 was just a nice pow ten number.